src/test/java/com/puppycrawl/tools/checkstyle/MainTest.java

```diff
@@ -78,60 +78,126 @@
-          + "  -b, --branch-matching-xpath=<xpath>%n"
-          + "                            Show Abstract Syntax Tree(AST) branches that match XPath%n"
-          + "  -c=<configurationFile>    Sets the check configuration file to use.%n"
-          + "  -C, --checker-threads-number=<checkerThreadsNumber>%n"
-          + "                            (experimental) The number of Checker threads (must be"
-          + " greater than zero)%n"
-          + "  -d, --debug               Print all debug logging of CheckStyle utility%n"
-          + "  -e, --exclude=<exclude>   Directory/File path to exclude from CheckStyle%n"
-          + "  -E, --executeIgnoredModules%n"
-          + "                            Allows ignored modules to be run.%n"
-          + "  -f=<format>               Sets the output format. Valid values: xml, plain."
-          + " Defaults to plain%n"
-          + "  -g, --generate-xpath-suppression%n"
-          + "                            Generates to output a suppression xml to use to suppress"
-          + " all violations%n"
-          + "                              from user's config%n"
-          + "  -h, --help                Show this help message and exit.%n"
-          + "  -j, --javadocTree         Print Parse tree of the Javadoc comment%n"
-          + "  -J, --treeWithJavadoc     Print full Abstract Syntax Tree of the file%n"
-          + "  -o=<outputPath>           Sets the output file. Defaults to stdout%n"
-          + "  -p=<propertiesFile>       Loads the properties file%n"
-          + "  -s=<suppressionLineColumnNumber>%n"
-          + "                            Print xpath suppressions at the file's line and column"
-          + " position.%n"
-          + "                              Argument is the line and column number (separated by"
-          + " a : ) in the%n"
-          + "                              file that the suppression should be generated for%n"
-          + "  -t, --tree                Print Abstract Syntax Tree(AST) of the file%n"
-          + "  -T, --treeWithComments    Print Abstract Syntax Tree(AST) of the file including"
-          + " comments%n"
-          + "  -V, --version             Print version information and exit.%n"
-          + "  -w, --tabWidth=<tabWidth> Sets the length of the tab character. Used only with"
-          + " \"-s\" option.%n"
-          + "                              Default value is 8%n"
-          + "  -W, --tree-walker-threads-number=<treeWalkerThreadsNumber>%n"
-          + "                            (experimental) The number of TreeWalker threads (must be"
-          + " greater than%n"
-          + "                              zero)%n"
-          + "  -x, --exclude-regexp=<excludeRegex>%n"
-          + "                            Regular expression of directory/file to exclude from"
-          + " CheckStyle%n");
+          "Usage: checkstyle [-dEghjJtTV] [-b=<xpath>] [-c=<configurationFile>] "
```

Comment:
Note that you change `%n` with `\n` in the test. Please use `%n` as the end-of-line marker to fix windows build.

Reply:
@pbludov Done